### PR TITLE
Add support for new rfid reader

### PR DESCRIPTION
**Pull request by hanswilw**
Changes code that handles reading cards to support new rfid readers.

### CODE REVIEW

**Comment by kotlarz** [comment]
Tested with 10+ cards, their data are unique. Registration of new cards and activation/deactivation is working fine. Only _issue_ is that we do not check a checksum (which has never been something we have done in the previous builds, so it is not really an issue per se).
OK to merge :+1:

File: chrome_app/main.js [hunk 1]
```diff
@@ -15,6 +15,35 @@ var updateTarget = function(value) {
...
+  '85': 'UNKNOWN COMMAND', // eslint-disable-line
+  '84': 'RESPONSE ERROR', // eslint-disable-line
+  '82': 'READER TIMEOUTE', // eslint-disable-line
+  '90': 'CARD DOES NOT SUPPORT THIS COMMAND', // eslint-disable-line
+  '8f': 'UNNSUPPORTED CARD IN NFC WRITE MODE' // eslint-disable-line
+};
+
+var onData = function(response) {
+    const hexValues = [];
+    for (let i = 0; i < response.length; i += 1) {
```
Comment (ekmartin) [inline chrome_app/main.js]: `const hexValues = response.map(res => res.toString(16));`?

**Comment by ekmartin** [comment]
What happened to the check sum code that @larseen wrote? https://github.com/webkom/vote/pull/224/commits/9a02a1f7deafbdfea706906873dd3c56ff7aa69a#diff-d9ca5ac5edc242800f590ce1f2f7fa92R58

If we have time it would probably be good to include.

**Comment by kotlarz** [comment]
@ekmartin It didn't work with the cards we are going to use, the university (NTNU) cards returns an ArrayBuffer of 22 items whilst the cards we are planning to use have an ArrayBuffer of 11 items. Unless I am missing something @larseen ?

If we have time it might be worth checking the datasheet on them and implement the checksum validation.

**Comment by larseen** [comment]
@kotlarz & @mrfarstad 
The code should work regardless of the message length.
I think it should pull out a message from the start flag `0xAA` to end flag `0xBB` rather than just assuming that a message is X long. This should be fixed since the data now assumed as a card rfid is  most probably just some message data and could even consist of multiple messages.

The readme should also be updated to include the driver needed to use the reader. Other than that it looks quite good 🙆‍♂️

**Comment by kotlarz** [comment]
@larseen I see what you mean now, and we saw that start and end flags when we were testing. I can fix it after Friday, but I'd rather not play around until after if that's okay with you?

**Comment by larseen** [comment]
Sure, Just create an issue so we remember it 👍 
I guess the most important thing is that we have something working now.

**Comment by hanswilw** [comment]
Closed in favor of #227

**Pull request closed without merging**